### PR TITLE
Some UX updates

### PR DESCRIPTION
**Pull request by meeow**
- Improved UX for small screens 
- Commented out some stuff that isn't worth the clutter in the UI for now
- Auto prompt user for their details upon first visit of a session id
- Save the previous username and autopopulate it next time
- Added password -> emoji mapping
- Increased color usage
- Too lazy to update the court section much for now

image

image

### CODE REVIEW

File: src/server/lib/credentials/google.ts [hunk 1]
```diff
@@ -4,5 +4,6 @@ export const GOOGLE = {
   credentials: JSON.parse(
     Buffer.from(process.env.GOOGLE_CREDENTIALS || '', 'base64').toString(),
   ) as CredentialBody,
-  projectId: 'prismatic-smoke-201521',
-};
+  projectId: process.env.PROJECT_ID || '',
```
Comment (meeow) [inline src/server/lib/credentials/google.ts:7]: @pohang you'll probably need to add project id and bucket name to the env file

**Review by pohang — APPROVED** [review]
cute

File: src/components/groups/AddPlayerDialog.tsx [hunk 2]
```diff
@@ -85,14 +90,28 @@ const AddPlayerDialog = (props: Props) => {
...
+  }
+
   return (
-    <Dialog open={open} onOpenChange={setOpen}>
+    <Dialog open={open || freshVisit} onOpenChange={onOpenChange}>
       <DialogTrigger asChild>
-        <Button>Add Player</Button>
+        <Button variant='outline'>
+          <div className='flex gap-1 items-center'>
+            <FontAwesomeIcon icon={faUser} />
```
Comment (pohang) [inline src/components/groups/AddPlayerDialog.tsx]: I don't think it's obvious that this is the button that you press to add a user.

Screenshot 2023-08-26 at 1 18 38 PM

Comment (pohang) [reply src/components/groups/AddPlayerDialog.tsx]: maybe toss a plus sign to the end or something?

Comment (meeow) [reply src/components/groups/AddPlayerDialog.tsx]: updated icon to user with plus sign

File: src/components/groups/PlayerList.tsx [hunk 3]
```diff
@@ -80,11 +79,15 @@ const PlayerList = (props: Props) => {
   const formattedPlayers = players.map((p) => {
     const signup = playerStatus.get(p.username.toLowerCase());
     let status;
+    let end = new Date(0);
 
     if (signup == null) {
       status = '🚫';
     } else {
       const { startsAt, endsAt, court } = signup;
+      if (endsAt) {
```
Comment (pohang) [inline src/components/groups/PlayerList.tsx:88]: when a user is on a reserved court, endsAt will be null and they will end up sorting with the users that need to be signed up. I addressed this in the past by setting the sort order to be a very large number.

**Review by pohang — APPROVED** [review]
left a comment about sort order, i can fix it too if you just want to merge.

**Comment by meeow** [comment]
> left a comment about sort order, i can fix it too if you just want to merge.

That'd be awesome 🏸

**Pull request merged**